test/duration_test.exs

```diff
@@ -111,5 +122,23 @@ defmodule DurationTests do
     assert d.megaseconds + d.seconds + d.microseconds > 0
   end
 
+  describe "now" do
+    test "now/0 returns the duration since epoch" do
+      erlang_now_microseconds = div(:os.system_time, 1000)
+      timex_now_microseconds = Timex.Duration.now() |> Timex.Duration.to_microseconds
+      difference_microseconds = timex_now_microseconds - erlang_now_microseconds
+      assert difference_microseconds < 1000000
+      assert difference_microseconds >= 0
+    end
```

Comment:
So I still don't love these tests, but I think that this is a better test than what I had as a doctest previously. Shouldn't be vulnerable to a race condition, but it could be vulnerable to something running exceptionally slow. The maximum microsecond_difference that I observed was 32, but most came in below 10. I would be extremely surprised if it ever came over 1,000,000 microseconds without something being actually wrong.